mu4e/mu4e-compose.el

```diff
@@ -514,7 +519,7 @@ automatically encrypt that reply."
 	  (encrypt (mml-secure-message-encrypt))
 	  (sign-and-encrypt (mml-secure-message-sign-encrypt)))))
 
-(defun* mu4e~compose-handler (compose-type &optional original-msg includes)
+(defun* mu4e~compose-handler (compose-type &optional original-msg includes as-attachment)
   "Create a new draft message, or open an existing one.
```

Comment:
In this one (and the other), don't pass 'as-attachment' as a parameter; rather, use the value of mu4e-compose-forward-as-attachment directly in the function, so only `mu4e~compose-handler` and `mu4e~draft-forward-construct` need changes.